src/kOS.Safe/Utilities/KOSNomenclature.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Text;
+using System.Reflection;
+using System.Collections.Generic;
+using System.Linq;
+using kOS.Safe.Encapsulation;
+using kOS.Safe.Exceptions;
+using kOS.Safe.Serialization;
+
+namespace kOS.Safe.Utilities
+{
+    /// <summary>
+    /// A class for the purpose of holding the mapping information that goes
+    /// back and forth between C# type names and thier KOS type name equivalents.
+    /// For the sake of this we are using unqualified C# names (without the namespace
```

Comment:
I need to edit this comment - this was true but isn't anymore.  Now it thinks in the fully qualified C# names.